test/InputSpec.js

```diff
@@ -51,7 +40,7 @@ describe('Input', function () {
     assert.equal(instance.getValue(), 'v');
   });
 
-  it('renders a submit button element when type=submit', function () {
+  it.skip('renders a submit button element when type=submit', function () {
```

Comment:
Ok I get it now, I'm fine with letting this through since the other PR will come along shortly.